### DISCUSSION TITLE
How do you Choose the Port?

### DISCUSSION THREAD

**Discussion by alex-fullam** [searxng/searxng#1455] [Q&A]
I'm running (well, TRYING TO RUN) searx on a raspberry pi with some other programs running on various ports. I can not find any way to choose the port. I've tried adding the full address in the .env file, and I also tried doing it in the settings file but I cannot figure out how to do it. How do you choose the port to make it actually run?

**Comment by colonical** [comment]
there are a lot of domain seller . if u add subdomain then u can use many servers with different subdomains on the same ip address.
so subdomain1.host server1
     subdomain2.host server2
etc...

**Comment by hirad1986** [comment]
What do you mean you can't choose a port? you can choose any port you want, as long as its not being used. Are you using docker? if so, the default is 8080:8080. If your port 8080 busy, choose a different one and replace it with the left one. for example 8880:8080 or 6060:8080 or any port you want.
